### DISCUSSION TITLE
Help: convert UTC time to system time

### DISCUSSION THREAD

**Discussion by ashithrahul** [moment/luxon#1209] [Q&A]
I am migrating the moment lib to luxon2.3 but am facing one issue with UTC time
I have a time expected to be in UTC and am trying to convert to my local time

```
const time = "2022-03-30T10:00:00";

console.log(moment.utc(time).local().format("lll"), "moment");
// expected output
// Mar 30, 2022 12:00 PM CEST
// Mar 30, 2022 3:30 PM IST
```

am not able to do the same in luxon.
can anyone point to me the correct section from the documentation? 

Thanks

**Comment by icambron** [comment]
https://moment.github.io/luxon/#/zones?id=system-zone-by-default

You just want

```js
DateTime.fromISO(time, { zone: "UTC" }).toLocal()
```
